api/api.go

```diff
@@ -193,7 +193,16 @@ func (api *Server) GetAccount(ctx context.Context, in *iotexapi.GetAccountReques
 		PendingNonce: pendingNonce,
 		NumActions:   numActions,
 	}
-	return &iotexapi.GetAccountResponse{AccountMeta: accountMeta}, nil
+	tipHeight := api.bc.TipHeight()
```

Comment:
can we get height from sf at very beginning of the function and use that as height of the block?